content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -279,7 +279,7 @@ rules:
 ```
 
 {{< note >}}
-You cannot restrict `create` or `deletecollection` requests by resourceName. For `create`, this
+You cannot restrict `create` or `deletecollection` or `list` requests by resourceName. For `create`, this
 limitation is because the object name is not known at authorization time.
 {{< /note >}}
```

Comment:
Ah yes. This has caught me out before!

I think I'd change upgrade this to a caution callout:
```suggestion
{{< caution >}}
You cannot restrict `create` or `deletecollection` or `list` requests by their resource name.

If you specify `resourceNames` along with the `list` verb, you are granting access to `get` all the resources
and not just the resource you named. This is because `list` returns a collection of all the matching resources.

For `create`, this limitation is because the name of the new object may not be known at authorization time.
{{< /caution >}}
```

Reply:
change note to caution